tests/libs/datasets/nytimes_anomalies_test.py

```diff
@@ -0,0 +1,123 @@
+import numpy as np
+from covidactnow.datapublic.common_fields import CommonFields
+
+from tests import test_helpers
+from tests.test_helpers import TimeseriesLiteral
+from libs.datasets import nytimes_anomalies
+from libs.datasets.taglib import TagType
+from libs.pipeline import Region
+
+
+def test_read_nytimes_anomalies():
+    nytimes_anomalies.read_nytimes_anomalies()
```

Comment:
Might be nice to make this an assert or assert that there are records returned

Reply:
Ah, yeah, good call. Done.